### ISSUE TITLE
bin/t-im should be deleted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3507]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)| @teratorn reported|
|-|-|
|Trac ID|trac#3507|
|Type|defect|
|Created|2008-10-27 22:08:32Z|
|Branch|https://github.com/twisted/twisted/tree/delete-t-im-3507|

t-im references twisted.scripts.im which doesn't exist anymore.

<details><summary>Searchable metadata</summary>

```
trac-id__3507 3507
type__defect defect
reporter__teratorn teratorn
priority__normal normal
milestone__ 
branch__branches_delete_t_im_3507 branches/delete-t-im-3507
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1225145312000000 1225145312000000
changetime__1225293328000000 1225293328000000
version__None None
owner__teratorn teratorn
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_1"></a>|@teratorn commented|
|-|-|

(In [25142]) Branching to 'delete-t-im-3507'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @teratorn|
|-|-|

You missed some parts.  Grep the repository for t-im.

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_4"></a>|@teratorn commented|
|-|-|

thanks, done now.

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)|@teratorn removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_6"></a>|@ralphm set owner to @teratorn|
|-|-|

I noticed that `twisted.words.scripts.im` does exist. I'm not sure about the usefulness of rewiring `t-im` back to it, though.

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_7"></a>|@teratorn removed owner|
|-|-|

bin/words/im is the binary that references twisted.words.scripts.im, so I don't think we need to worry about bin/t-im... presumably it has been broken for years, but I don't know. trac failed to tell me when twisted.scripts.im was last seen.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set owner to @teratorn|
|-|-|

Looks good, please merge.  Thanks.

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_9"></a>|@teratorn set status to `closed`|
|-|-|

(In [25178]) merge delete-t-im-3507
Author: teratorn
Reviewer: exarkun
Fixes: [#3507](https://github.com/twisted/twisted/issues/3507)
